dbld/images/packages/packages.txt

```diff
@@ -0,0 +1,171 @@
+#autogen
```

Comment:
since this has become a simple file now, I'd omit the directory and move it to the images/ subfolder.